packages/components/psammead-navigation/README.md

```diff
@@ -206,6 +205,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
   dir={dir}
 />
 ```
+Note that in order for the `AmpMenuButton` toggling to work correctly, an `id` should be added to the `Navigation` component in Psammead. This `id` can be passed in as a prop to the component.
```

Comment:
Sorry if I am misunderstanding, but did you mean in Simorgh not Psammead?

Reply:
I meant Psammead. I am referring to the addition of an `id` to the Navigation help with the toggling of the classname. 

I can rephrase this to try and make it more clear that it's the class toggling functionality that won't work correctly

Reply:
Or maybe I should remove the comment ? As the addition of the id was more of a fix

Reply:
No, I think that the comment should be there. I just wanted to clarify about the id because we are now spreading props in the Navigation. All good :) Thank you

Reply:
Made this(https://github.com/bbc/psammead/pull/2802#discussion_r356530924) change for it to be more clear